src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -1834,7 +1834,7 @@ void Unit::CalcAbsorbResist(DamageInfo& damageInfo)
         absorbAurEff->GetBase()->CallScriptEffectAfterAbsorbHandlers(absorbAurEff, aurApp, damageInfo, tempAbsorb);
 
         // Check if our aura is using amount to count damage
-        if (absorbAurEff->GetAmount() >= 0)
+        if (absorbAurEff->GetAmount() > 0)
```

Comment:
Why change this?

Reply:
When the amount is 0 and the spell is passive it will still remove, although it shouldn't.
Tested on brambles, 203953.